src/main/resources/interface-specification.yml

```
@@ -817,6 +817,18 @@ components:
           maxLength: 64
       required:
         - code
+    CompactTenant:
+      description: Tenant Compact model
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        externalId:
+          $ref: '#/components/schemas/ExternalId'  
+      required:
+        - id
+        - externalId
```

as discussed, we can just return the id, like
```suggestion
    ResourceId:
      description: Modified resource id
      type: object
      properties:
        id:
          type: string
          format: uuid
      required:
        - id
```